connman/src/network.c

```diff
@@ -626,8 +629,17 @@ static void set_connected(struct connman_network *network)
 	DBG("service %p ipv4 %p ipv6 %p", service, ipconfig_ipv4,
 		ipconfig_ipv6);
 
-	__connman_network_enable_ipconfig(network, ipconfig_ipv4);
-	__connman_network_enable_ipconfig(network, ipconfig_ipv6);
+	ipv4_err = __connman_network_enable_ipconfig(network, ipconfig_ipv4);
+	ipv6_err = __connman_network_enable_ipconfig(network, ipconfig_ipv6);
+
+	if (!network->require_all_ip_types) {
+		if (ipv4_err & ipv6_err)
```

Comment:
Probably meant to use && here?

Reply:
Yeah! Good oops.

Reply:
Included this in the network.c commit 5e2cd5dbb359f5ce752ef51ac89ae1eb594962ef